packages/app/src/background/shared/__tests__/utils.test.ts

```diff
@@ -1,4 +1,32 @@
-import { deferredPromise } from "../utils";
+/**
+ * @jest-environment jsdom
+ */
+import chrome from "jest-chrome";
+
+import {
+  BROWSER_PLATFORM_BRAVE,
+  BROWSER_PLATFORM_CHROME,
+  BROWSER_PLATFORM_EDGE,
+  BROWSER_PLATFORM_FIREFOX,
+  BROWSER_PLATFORM_OPERA,
+} from "@src/constants";
+
+import { createChromeOffscreen, deferredPromise, getBrowserPlatform } from "../utils";
+
+Object.assign(global, chrome);
+
+// Mock the chrome module
+jest.mock("jest-chrome", () => ({
```

Comment:
Why do you mock dev dependency? All mocks for browser related things are in `src/setupTests.ts`.

Reply:
Because I want to mock only `chrome.*` APIs specifically, and since `jest-chrome` APIs don't have `chrome.offscreen.*` APIs I had to mock those APIs.

Reply:
Yes, this is exactly what I mean. You can take a look at `setupTests.ts` and take a look how browser is mocked.
So the same will be for chrome.

```

Object.defineProperty(global, "chrome", {
  offscreen: {
        hasDocument: jest.fn(),
        createDocument: jest.fn(),
        Reason: jest.fn(() => ({
          DOM_SCRAPING: "DOM_SCRAPING",
        })),
      },
});
```